easybuild/easyconfigs/g/GCCcore/GCCcore-5.x-asan-fix-missing-include-signal-h.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/libsanitizer/asan/asan_linux.cc b/libsanitizer/asan/asan_linux.cc
```

Comment:
@zao Please mention upstream source or include author + include short description of what this fixes (bonus points if you include the exact error message)